core/src/main/java/hudson/security/csrf/CrumbIssuerDescriptor.java

```diff
@@ -64,7 +64,7 @@ public String getCrumbRequestField() {
      */
     public void setCrumbRequestField(String requestField) {
         if (Util.fixEmptyAndTrim(requestField) == null) {
-            crumbRequestField = ".crumb";
+            crumbRequestField = CrumbIssuer.DEFAULT_CRUMB_NAME;
```

Comment:
Weird, seems like all this stuff should really have been in `DefaultCrumbIssuer.DescriptorImpl`, not in the supertype.